src/main/java/wolox/gmjavatraining/GmJavaTrainingApplication.java

```diff
@@ -5,8 +5,8 @@
 import org.springframework.boot.autoconfigure.domain.EntityScan;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
-@EnableJpaRepositories("wolox.gmjavatraining.persistence.repo")
-@EntityScan("wolox.gmjavatraining.persistence.model")
+@EnableJpaRepositories(basePackages = "wolox.gmjavatraining.repositories")
```

Comment:
These two annotations need to get removed, only needed when your files are outside of the default java project structure (src/main/java, src/main/resources, src/test/java, src/test/resources)